Sources/PointFree/PageLayout.swift

```diff
@@ -163,10 +163,33 @@ private func flashClass(for priority: Flash.Priority) -> CssSelector {
   }
 }
 
-private let highlightJsHead: [ChildOf<Element.Head>] = [
-  link(
-    [rel(.stylesheet), href("//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css")]
-  ),
-  script([src("//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js")]),
-  script("hljs.initHighlightingOnLoad();")
+private let prismJsHead: [ChildOf<Element.Head>] = [
+  script([src("//cdnjs.cloudflare.com/ajax/libs/prism/1.10.0/prism.min.js")]),
+  script(
+    """
+    Prism.languages.swift = Prism.languages.extend("clike", {
```

Comment:
I just grabbed their current plugin and modified it since it was basically working in a Swift 2 world. Made minimal modifications so far but we can tweak further as we need to.